app/views/partials/GenericIneligiblePartial.scala.html

```diff
@@ -33,4 +33,4 @@
 }
 </p>
 
-<p id="call-us-content" class="govuk-body">@{Html(Messages.NotEligible.`Call us on 0300 123 1813 as you may be able to set up a plan over the phone`.show)}</p>
+<p id="call-us-content" class="govuk-body">@{Html(Messages.NotEligible.`Call us on 0300 322 7835 as you may be able to set up a plan over the phone`.show)}</p>
```

Comment:
should the change in phone number be for simple assessment only or is it for all regimes here?